native/src/components/ExportEventButton.tsx

```
@@ -39,16 +45,39 @@ const ExportEventButton = ({ event }: ExportEventButtonType): ReactElement => {
       endDate = event.date.endDate.plus({ minutes: 1 }).toFormat("yyyy-LL-dd'T'00:00:00.000'Z'")
     }
 
+    let eventOptions: CalendarEventWritable = {
+      startDate,
+      endDate,
+      allDay,
+      calendarId,
+      location: event.location?.fullAddress,
+      description: event.excerpt, // Android
+      notes: event.excerpt, // iOS
+    }
+
+    if (exportAll && event.date.recurrenceRule) {
+      // @ts-expect-error Users don't need to set a last date or a number of occurrences in the CMS
```

i think its cleaner to override the interface that suppress the error